public/css/vendor/normalize.css

```diff
@@ -0,0 +1,349 @@
+/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
```

Comment:
My feelings about normalizers are that, by and large, we should incorporate them in our code. That is, instead of resetting to a sane default and then over-riding for our site's needs, we should set to out site's needs right off the bat.

That said, I don't see a lot of overlap between what is in this reset and what we define ourselves, but at any point that we find ourselves overriding the reset, we should fork it and make the reset the basis of our styling. A good example of that is here: https://igoradamenko.github.io/awsm.css/

No change needed, and if you disagree with the above, please let me know, I don't want to dictate how things are done.